<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QSerialPortDlg</class>
 <widget class="QWidget" name="QSerialPortDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btnFind">
        <property name="text">
         <string>FindPort</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cbxPortName"/>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpen">
        <property name="text">
         <string>OpenPort</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSend">
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="lblSendArea">
        <property name="text">
         <string>lblSendArea</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="editSendArea"/>
      </item>
      <item>
       <widget class="QLabel" name="lblReceiveArea">
        <property name="text">
         <string>lblReceiveArea</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="editReceiveArea"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
